data/README.md

```diff
@@ -111,3 +111,135 @@ for f in *.txt; do cat $f | perl -pe 's/\t.*//' > $f.new; mv $f.new $f; done
 cd publications
 for f in *.txt; do cat $f | perl -pe 's/^([^\d]+)\s?,\d+;/$1/' > $f.new; mv $f.new $f; done
 ```
+
+## To create pre-version of list of publishers of FIS data:
+Read in `Publikationen.csv` and `Personen_Einrichtungen_2023_06.csv` from 
+`/data/FIS` and change `Publikationen.csv` by adding the following columns
+* `lastname` of the person from `Personen_Einrichtungen_2023_06.csv`
+* `forename` of the person from `Personen_Einrichtungen_2023_06.csv`
+* `titles` that contains all publications that are listed in this dataframe for 
+the respective person 
+* `institution` that contains only the part of the description of the 
+institution in `Publikationen.csv` up to the first `/`
+
+[//]: # (Hello)
```

Comment:
What is this?
You get a new line by appending two spaced at the end of the previous line.